week03/assignment03.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import "fmt"
+
+const ErrMovieNotPlayed = MovieError("can't review a movie without watching it first")
+
+type MovieError string
+
+func (e MovieError) Error() string {
+	return string(e)
+}
```

Comment:
While I understand why you went this route for your errors, it is a bit overkill for what we need here. :)

We will cover errors in depth in week 5. 

For now if you want to dry up your code, I would recommend the following:

```suggestion
var errMovieNotPlayed = fmt.Errorf("can't review a movie without watching it first")
```